### ISSUE TITLE
Discussion Place?

### ISSUE THREAD

**Issue by inchingforward** [OvermindDL1/bucklescript-tea#54]
I apologize for using an issue for this...feel free to close this.

I don't remember where I first heard about this project, but searching led me to the [Elixir Forum](https://elixirforum.com/t/bucklescript/1579) post, the bucklescript channel in Discord, and your blog posts.

I've considered writing blog posts about Bucklescript-TEA, writing some tutorials, or contributing to documentation.  Would having a dedicated place to ask questions, gauge interest in topics, etc, be worthwhile?  Do any of the above-mentioned places (or this one) make sense for discussing Bucklescript-TEA?

**Comment by OvermindDL1** [comment]
Ooo that'd be awesome!

Speaking on the Elixir Forum would be most likely where I'd see and respond the quickest.

For real-time chat the bucklescript channel on the reasoml Discord would be the best (ping me repeatedly ^.^).  :-)

**Comment by quernd** [comment]
As the year is coming to an end, I want to say "thank you" for this awesome project, and give back to the community.  I have some free time this holiday season and have started working on a lengthy chess-themed tutorial on writing an SPA with Bucklescript-TEA.  If things go well, I will post one part of the tutorial for each of the "12 Days of Christmas" from today through January 5.

Here's the code: https://github.com/quernd/tea-chess
Here's the writeup: https://quernd.github.io/tutorials/tea-chess

I've uploaded today's post (and tentatively two more parts, but I'll keep refining them as I go).  Enjoy!

**Comment by jackalcooper** [comment]
Cool!

**Comment by OvermindDL1** [comment]
> As the year is coming to an end, I want to say "thank you" for this awesome project, and give back to the community. I have some free time this holiday season and have started working on a lengthy chess-themed tutorial on writing an SPA with Bucklescript-TEA. If things go well, I will post one part of the tutorial for each of the "12 Days of Christmas" from today through January 5.

Hah, that is really cool!  I may link to it when I do some updating if that's fine?  :-)

**Comment by quernd** [comment]
> Hah, that is really cool! I may link to it when I do some updating if that's fine? :-)

Yes of course you may.  I'm also thinking of other places to post an announcement, what do you guys think, should I wait for when the entire thing is complete?  I was thinking /r/ocaml, discuss.ocaml.org and maybe the Elixir Forum thread mentioned above, but I haven't really been very extrovert on the web so I'm not sure what's appropriate.

**Comment by jackalcooper** [comment]
@quernd would you make it based on Elm tutorial or you are gonna write it from scratch? I mean we kind of lack a proper getting started guide here. Maybe a helpful getting started guide will attract more people using bs-tea.

**Comment by quernd** [comment]
@jackalcooper I'm writing it from scratch – it's more or less a writeup of my own experience working on a small project. The first part explains how to get started using https://github.com/tcoopman/bucklescript-tea-starter-kit and the basics of the Elm architecture, but it's a little faster paced than the Elm tutorial I think.

**Comment by OvermindDL1** [comment]
I'd say add links early, then just post updates as updates come out.  :-)

**Comment by quernd** [comment]
Thanks guys.  8 parts out of 12 are now online, and it's starting to look like an SPA :-)

* Someone featured me on [/r/altjs](https://www.reddit.com/r/altjs/comments/7me6gq/a_chessthemed_bucklescripttea_spa_tutorial/) which I crossposted to [/r/ocaml](https://www.reddit.com/r/ocaml/comments/7n4nqx/a_chessthemed_bucklescripttea_spa_tutorial/)
* Also see [Elixir Forum](https://elixirforum.com/t/bucklescript/1579/139) and [discuss.ocaml.org](https://discuss.ocaml.org/t/about-bucklescript/178/14)

**Comment by martindemello** [comment]
i also posted to lobsters :)
https://lobste.rs/s/e4ps7y/chess_themed_bucklescript_tea_tutorial

**Comment by neochrome** [comment]
Maybe we could create a sub redit like `/r/bucklescript-tea` ?

**Comment by OvermindDL1** [comment]
@neochrome That then becomes very permanent of a name then especially as I am planning to expand this, hmm...

**Comment by neochrome** [comment]
I guess another viable option could be to use issues here at github for discussions, just labelling them as such...

**Comment by OvermindDL1** [comment]
> I guess another viable option could be to use issues here at github for discussions, just labelling them as such...

I'm always good with that, though if real-time is wanted then IRC is usually best.  ^.^
